android-components/components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/SuggestionFetcher.kt

```diff
@@ -113,6 +113,10 @@ internal class SuggestionFetcher(
         updatedSuggestions.addAll(suggestions)
         updatedSuggestions.sortByDescending { suggestion -> suggestion.score }
 
+        if (updatedSuggestions.isNotEmpty()) {
+            group.priority = updatedSuggestions[0].score
```

Comment:
Should be check that this index is accessible before we try too access it. For example maybe this list could be empty, and we could cause an exception by accessing it.